js/data.js

```diff
@@ -72,31 +43,39 @@ const PHOTOS_ARRAY = [
 ];
 
 const MAX_PRICE = 100000;
-
 const MAX_ROOMS = 5;
-
 const MAX_GUESTS = 10;
 
-const getAddressElement = (array) => {
-  const randomElement = getRandomArrayElement(array);
-  return ((randomElement.lat).toString()).concat(', ', ((randomElement.lng).toString()));
+const offerElement = (_elem, id) => {
+  const lat = getRandomFloat(LAT_NUMBER_MIN, LAT_NUMBER_MAX, NUMBER_OF_DECIMALS);
+  const lng = getRandomFloat(LNG_NUMBER_MIN, LNG_NUMBER_MAX, NUMBER_OF_DECIMALS);
+
+  return {
+    author: {
+      avatar: `img/avatars/user${String(++id).padStart(2, '0')}.png`,
+    },
+
+    offer: {
+      title: 'Обьявление о сдаче жилья',
+      address: String(`${lat },${  lng}`),
```

Comment:
```suggestion
      address: `${lat }, ${  lng}`,
```
ПРиведение лишнее, у тебя и так строка